stdlib/builtins.pyi

```diff
@@ -1018,7 +1018,7 @@ if sys.version_info >= (3, 8):
         optimize: int = ...,
         *,
         _feature_version: int = ...,
-    ) -> Any: ...
+    ) -> object: ...
```

Comment:
This one I think should also remain Any; it has a more precise return type that depends on the arguments.

It can perhaps be expressed with overloads but probably we haven't done it because this function is rarely used and so nobody cares enough.

Reply:
Cool, I'll revert